File: Challenges/Ruby/fractionReducing.rb
def fractionReducing(fraction)
    res = []
    
    (1...10).each do |i|
        if fraction[0] % i == 0 and fraction[1] % i == 0 then
            res[0] = fraction[0] / i
            res[1] = fraction[1] / i
        end
    end
        
    return res
end

File: Challenges/Java/FractionReducing.java
final class FractionReducing {

	int[] fractionReducing(int[] fraction) {
	    int[] res = new int[2];
	    
	    for(int i = 1; i < 10; i++) {
	        if(fraction[0] % i == 0 && fraction[1] % i == 0) {
	            res[0] = fraction[0] / i;
	            res[1] = fraction[1] / i;
	        }
	    }
	    
	    return res;
	}
}	